docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -151,16 +151,17 @@ Will return a JSON object of the form:
 {
   "Images" : [
     {
-      "Candidate": "123456",
-      "PSCID": "MTL001",
-      "Visit": "V1",
-      "Visit_date": "2016-08-09", /* The date of the session. This will be null for phantoms and session that are not yet started */
-      "Site": "Montreal Neurological Institute",
-      "ScanType": "t2", /* Acquisition protocol */
-      "QC_status": "Pass|Fail|null",
-      "Selected": "true|false|null",
-      "Link": "\/candidates\/300022\/V1\/images\/loris-MRI_123456_V1_t2_001.mnc", /* URL relative to this API */
-      "InsertTime": "2016-08-09T14:15:30-05:00" /* The inserted date ISO 8601 */
+      "Candidate": "400168",
+      "File": "assembly/400168/V2/mri/native/demo_400168_V2_t1_001.mnc",
```

Comment:
I don't see why File under the assembly directory is part of the result returned from the API. What can you use this for that you can't use `Link` for? This seems like it's primarily intended to make it easier to bypass the API.

Reply:
@xlecours ?

Reply:
I agree, it is something internal to LORIS and should probably not be exposed.
I think it is something I added while migrating the endpoints to modules/api.
@cmadjar Do you remember something using this ?

Reply:
hehehe, it is very rare that a sentence starting with "@cmadjar Do you remember..." leads to something useful since I have the memory of a red fish. ;)

Honestly I don't remember and have no idea. Sorry :S

Reply:
Fun of the day. How comes that a question to Xavier needs to ask cmadjar.

Reply:
So can we unexpose it instead of changing the spec to add it?